blob/src/test/java/io/crate/integrationtests/BlobIntegrationTest.java

```diff
@@ -305,4 +310,37 @@ public void testIndexOnNonBlobTable() throws IOException {
             is("{\"_index\":\"test_no_blobs\",\"_type\":\"default\"," +
                "\"_id\":\"1\",\"_version\":1,\"_shards\":{\"total\":1,\"successful\":1,\"failed\":0},\"created\":true}"));
     }
+
+    @Test
+    public void testBlobShardStatsAreUpdatedOnUploadAndDelete() throws IOException {
```

Comment:
pls check if there is a tests for incremental stats updates which adds a blob with the same digest twice and checks stats. also for deletion (deleting twice).

pls add if not already there (could also be an integration test)